src/app/pages/ArticlePage/Byline/README.md

```diff
@@ -0,0 +1,34 @@
+# Byline
+
+## Description
+
+A Byline section with all relevant article's author informations.
+
+## Props
+
+| Name     | type     | Description                                                                              |
+| -------- | -------- | ---------------------------------------------------------------------------------------- |
+| blocks   | object   | byline type content                                                                      |
+| children | JSX Node | used for passing the Timestamp component as a listItem on the article information region |
+
+## Example ltr/rtl
+
```

Comment:
Empty section here

Reply:
Oh, no I just messed up the headings. The # Without Timestamp and # With Timestamp should be level 3 not level 1.
Thanks for noticing it, I will fix this now